jdi-light-vuetify/src/main/java/com/epam/jdi/light/vuetify/asserts/MenuAssert.java

```diff
@@ -37,6 +49,12 @@ public MenuAssert optionsTitles(List<String> optionsTitles) {
         return this;
     }
 
+    @JDIAction("Assert that '{name}' has expected options")
+    public MenuAssert optionsList(List<String> optionsTitles) {
```

Comment:
It is very difficult to use if we have optionsList in validation and getOptions in element. They must have similar names

Reply:
Modified the names.